### ISSUE TITLE
uv commands giving error for G1 LocoManipulation whole body

### ISSUE THREAD

**Issue by ANUBHAV2142** [NVIDIA/Isaac-GR00T#509]
### 🐛 Describe the bug

- [x] I am trying to run the basic inference script with uv command as mentioned in the documentation:

> uv run python scripts/deployment/standalone_inference_script.py \
>   --model-path nvidia/GR00T-N1.6-3B \
>   --dataset-path demo_data/gr1.PickNPlace \
>   --embodiment-tag GR1 \
>   --traj-ids 0 1 2 \
>   --inference-mode pytorch \
>   --action-horizon 8

**Which is giving me hugging face https connection pool SSL error.**

<img width="1364" height="389" alt="Image" src="https://github.com/user-attachments/assets/a5728063-935c-4fb3-841d-b8975af084bb" />

- [x] Similarly, I am trying the G1 LocoManipulation whole body Option 2: Remote finetuned checkpoint (directly runnable):

> uv run python gr00t/eval/run_gr00t_server.py \
>     --model-path nvidia/GR00T-N1.6-G1-PnPAppleToPlate \
>     --embodiment-tag UNITREE_G1 \
>     --use-sim-policy-wrapper

I need to know if instead of using uv commands for the inferencing and finetuning, will pip/python command work instead? 
If so, then how will these commands change?

I am using linux machine with cuda_12.4.r12.4.

Thanks

### Versions

Python 3.10.18
absl-py==2.3.1
accelerate==1.2.1
aiohappyeyeballs==2.6.1
aiohttp==3.13.3
aiosignal==1.4.0
albucore==0.0.17
albumentations==1.4.18
annotated-types==0.7.0
antlr4-python3-runtime==4.9.3
anyio==4.11.0
anytree==2.13.0
asttokens==3.0.0
astunparse==1.6.3
async-timeout==5.0.1
attrs==25.4.0
av==15.0.0
blessings==1.7
cachetools==6.2.1
certifi==2026.1.4
charset-normalizer==3.4.3
click==8.1.8
cloudpickle==3.1.1
cmeel==0.57.3
cmeel-assimp==5.4.3.1
cmeel-boost==1.83.0
cmeel-console-bridge==1.0.2.3
cmeel-octomap==1.10.0
cmeel-qhull==8.0.2.1
cmeel-tinyxml==2.6.2.3
cmeel-urdfdom==3.1.1.1
cmeel-zlib==1.3.1
comm==0.2.3
contourpy==1.3.2
cramjam==2.11.0
cycler==0.12.1
datasets==3.6.0
decorator==5.2.1
decord==0.6.0
deepspeed==0.17.6
dex_retargeting==0.4.6
diffusers==0.35.1
dill==0.3.8
dm-tree==0.1.8
docker-pycreds==0.4.0
docstring_parser==0.17.0
egl_probe==1.0.2
eigenpy==3.5.1
einops==0.8.1
eval_type_backport==0.2.2
exceptiongroup==1.3.0
executing==2.2.1
Farama-Notifications==0.0.4
fastparquet==2024.11.0
filelock==3.20.0
flaky==3.8.1
flash_attn==2.7.4.post1
flatbuffers==25.9.23
flatdict==4.0.1
fonttools==4.60.1
frozenlist==1.8.0
fsspec==2025.3.0
fvcore==0.1.5.post20221221
gast==0.6.0
gitdb==4.0.12
GitPython==3.1.45
google-auth==2.41.1
google-auth-oauthlib==1.2.2
google-pasta==0.2.0
-e git+https://github.com/NVIDIA/Isaac-GR00T@e8e625f4f21898c506a1d8f7d20a289c97a52acf#egg=gr00t
grpcio==1.75.1
gym==0.23.1
gym-notices==0.1.0
gymnasium==1.2.2
h5py==3.12.1
hf-xet==1.1.10
hidapi==0.14.0.post2
hjson==3.1.0
hpp-fcl==2.4.4
huggingface-hub==0.35.3
hydra-core==1.3.2
idna==3.11
imageio==2.34.2
imageio-ffmpeg==0.6.0
importlib_metadata==8.7.0
iniconfig==2.1.0
iopath==0.1.10
ipython==8.37.0
ipywidgets==8.1.5
-e git+https://github.com/isaac-sim/IsaacLab.git@e3f43d9ea3a7375e54e0f8253897d318ee8d0367#egg=isaaclab&subdirectory=source/isaaclab
-e git+https://github.com/isaac-sim/IsaacLab.git@e3f43d9ea3a7375e54e0f8253897d318ee8d0367#egg=isaaclab_assets&subdirectory=source/isaaclab_assets
-e git+https://github.com/isaac-sim/IsaacLab.git@e3f43d9ea3a7375e54e0f8253897d318ee8d0367#egg=isaaclab_mimic&subdirectory=source/isaaclab_mimic
-e git+https://github.com/isaac-sim/IsaacLab.git@e3f43d9ea3a7375e54e0f8253897d318ee8d0367#egg=isaaclab_rl&subdirectory=source/isaaclab_rl
-e git+https://github.com/isaac-sim/IsaacLab.git@e3f43d9ea3a7375e54e0f8253897d318ee8d0367#egg=isaaclab_tasks&subdirectory=source/isaaclab_tasks
jedi==0.19.2
Jinja2==3.1.6
joblib==1.5.2
json-numpy==2.1.1
jsonschema==4.25.1
jsonschema-specifications==2025.9.1
junitparser==4.0.2
jupyterlab_widgets==3.0.15
keras==2.15.0
kiwisolver==1.4.9
kornia==0.7.4
kornia_rs==0.1.9
lazy_loader==0.4
libclang==18.1.1
llvmlite==0.45.1
lmdb==1.7.5
loop-rate-limiters==1.2.0
lxml==6.0.2
Markdown==3.9
markdown-it-py==4.0.0
MarkupSafe==3.0.3
matplotlib==3.10.1
matplotlib-inline==0.2.1
mdurl==0.1.2
ml_dtypes==0.5.3
moviepy==2.2.1
mpmath==1.3.0
msgpack==1.1.0
msgpack-numpy==0.4.8
multidict==6.7.0
multiprocess==0.70.16
networkx==3.4.2
ninja==1.13.0
nlopt==2.7.1
numba==0.62.1
numpy==1.26.4
numpydantic==1.6.7
nvidia-cublas-cu12==12.8.3.14
nvidia-cuda-cupti-cu12==12.8.57
nvidia-cuda-nvrtc-cu12==12.8.61
nvidia-cuda-runtime-cu12==12.8.57
nvidia-cudnn-cu12==9.7.1.26
nvidia-cufft-cu12==11.3.3.41
nvidia-cufile-cu12==1.13.0.11
nvidia-curand-cu12==10.3.9.55
nvidia-cusolver-cu12==11.7.2.55
nvidia-cusparse-cu12==12.5.7.53
nvidia-cusparselt-cu12==0.6.3
nvidia-nccl-cu12==2.26.2
nvidia-nvjitlink-cu12==12.8.61
nvidia-nvtx-cu12==12.8.55
oauthlib==3.3.1
omegaconf==2.3.0
onnx==1.19.1
opencv-python==4.8.0.74
opencv-python-headless==4.11.0.86
opt_einsum==3.4.0
orjson==3.11.4
packaging==23.2
pandas==2.2.3
parso==0.8.5
peft==0.17.1
pettingzoo==1.25.0
pexpect==4.9.0
pillow==11.2.1
pin==2.7.0
pin-pink==3.1.0
pipablepytorch3d==0.7.6
platformdirs==4.5.0
pluggy==1.6.0
portalocker==3.2.0
prettytable==3.3.0
proglog==0.1.12
prompt_toolkit==3.0.52
propcache==0.4.1
protobuf==5.29.5
psutil==5.9.8
ptyprocess==0.7.0
pure_eval==0.2.3
py-cpuinfo==9.0.0
pyarrow==23.0.0
pyasn1==0.6.1
pyasn1_modules==0.4.2
pydantic==2.10.6
pydantic_core==2.27.2
pyglet==1.5.31
Pygments==2.19.2
pyparsing==3.2.5
pytest==8.4.2
pytest-mock==3.15.1
python-dateutil==2.9.0.post0
python-dotenv==1.2.1
pytransform3d==3.14.3
pytz==2025.2
PyYAML==6.0.2
pyzmq==27.0.1
qpsolvers==4.8.1
quadprog==0.1.13
ray==2.51.0
referencing==0.37.0
regex==2025.9.18
requests==2.32.3
requests-oauthlib==2.0.0
rich==14.2.0
rl_games @ git+https://github.com/isaac-sim/rl_games.git@6b3534f29568158e9e29ec8bf83cc88fce5f0cae
robomimic @ git+https://github.com/ARISE-Initiative/robomimic.git@7c66e7a41b5d9dcc905b1a68346bfee1b49b79c9
rpds-py==0.27.1
rsa==4.9.1
rsl-rl-lib==3.0.1
safetensors==0.6.2
scikit-image==0.25.2
scikit-learn==1.7.2
scipy==1.15.3
sentry-sdk==2.41.0
setproctitle==1.3.7
shtab==1.7.2
six==1.17.0
skrl==1.4.3
smmap==5.0.2
sniffio==1.3.1
stable_baselines3==2.7.0
stack-data==0.6.3
starlette==0.45.3
sympy==1.14.0
tabulate==0.9.0
tensorboard==2.15.2
tensorboard-data-server==0.7.2
tensorboardX==2.6.4
tensordict==0.9.0
tensorflow==2.15.0
tensorflow-estimator==2.15.0
tensorflow-io-gcs-filesystem==0.37.1
termcolor==3.2.0
threadpoolctl==3.6.0
tianshou==0.5.1
tifffile==2025.5.10
timm==1.0.14
tokenizers==0.21.4
toml==0.10.2
tomli==2.3.0
torch==2.7.0+cu128
torchcodec==0.4.0
torchvision==0.22.0+cu128
tqdm==4.67.1
traitlets==5.14.3
transformers==4.51.3
trimesh==4.9.0
triton==3.3.0
typeguard==4.4.2
typing_extensions==4.12.2
tyro==0.9.17
tzdata==2025.2
urllib3==2.5.0
uv==0.9.26
wandb==0.23.0
warp-lang==1.9.1
wcwidth==0.2.14
Werkzeug==3.1.3
widgetsnbextension==4.0.14
wrapt==1.14.2
xxhash==3.6.0
yacs==0.1.8
yarl==1.22.0
zipp==3.23.0

**Comment by xiaotongc0** [comment]
This seems to be a connection / certification error when downloading the model from huggingface. Could you please share your environment setting, e.g. have you used huggingface-cli to login with write token? After you passed that part, you may refer to #554 if seeing similar problems. Basically, you would need to run inference from the fine-tuned checkpoint `nvidia/GR00T-N1.6-G1-PnPAppleToPlate` than the base model, or do fine-tuning locally before inference.
